src/components/TopBar/TopBar.vue

```diff
@@ -299,6 +305,7 @@ export default {
 
 		showUpcomingEvent() {
 			return this.nextEvent && !this.isInCall && !this.isSidebar && !this.isMobile
+				&& this.conversation.type === CONVERSATION.TYPE.NOTE_TO_SELF
 		},
```

Comment:
Seems to be unrelated and breaking upcoming events @DorraJaouad

Reply:
Or should be `!==`

Reply:
> Or supposed to be `!==`

Still not related to task counting though 👀

Reply:
Sorry, it should be `!==` indeed

Reply:
> > Or supposed to be `!==`
> 
> Still not related to task counting though 👀

Also true, but as I was adding the counter and touching that space in `TopBar`, I thought it could be slipped in this PR.